modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/rebalance/DistributionZoneRebalanceEngineTest.java

```diff
@@ -384,8 +384,10 @@ void replicasTriggersAssignmentsChangingOnNonDefaultZones() throws Exception {
 
         when(distributionZoneManager.dataNodes(anyLong(), anyInt(), anyInt())).thenReturn(completedFuture(Set.of("node0")));
 
+        byte[] assignmentsBytes = Assignments.toBytes(Set.of(Assignment.forPeer("node0")));
```

Comment:
Should be `Assignments.of(Assignment.of("node0"));`, don't you think?

Reply:
Yes, why not